packages/@dynamic-labs-connectors/abstract-global-wallet-evm/src/AbstractEvmWalletConnector.ts

```diff
@@ -0,0 +1,107 @@
+import { type EthereumWalletConnectorOpts } from '@dynamic-labs/ethereum-core';
+import { EthereumInjectedConnector, type IEthereum } from '@dynamic-labs/ethereum';
+import { toPrivyWalletProvider } from '@privy-io/cross-app-connect'
+import { transformEIP1193Provider } from '@abstract-foundation/agw-client';
+import { abstractTestnet } from 'viem/chains';
+import { DynamicError } from '@dynamic-labs/utils';
+import { type Chain, logger } from '@dynamic-labs/wallet-connector-core';
+import { findWalletBookWallet } from '@dynamic-labs/wallet-book';
+import { toHex } from 'viem';
+const AGW_APP_ID = 'cm04asygd041fmry9zmcyn5o5';
```

Comment:
Is it ok to hardcode this?

Reply:
Yes - this is a public app id used to identify us in the cross app connection process.